### PR TITLE
Full ko_KR translation

### PR DESCRIPTION
**Pull request by infopcgood**
Translated the full site into Korean. Translation might have some inconsistency&problems, but I plan on updating and maintaining the translation everytime I see something wrongly translated on the site.

### CODE REVIEW

File: .gitignore
Comment (bdefore) [inline .gitignore:1]: please revert this change. it's unwanted to ignore this file when using yarn, which this project uses by default.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
-# protondb-i18n
+# protondb-i18n-ko_KR
 
-A crowdsourced journey to translate [ProtonDB](https://www.protondb.com) ([News post](https://www.protondb.com/news/protondb-spricht-nun-auch-deutsch))
+[ProtonDB](https://www.protondb.com) 사이트에 대한 한국어 번역본입니다.
```
Comment (bdefore) [inline README.md:3]: we have a standard place for attribution in a section below this, please add yours according to convention, or i can add for you: https://github.com/bdefore/protondb-i18n#hall-of-fame

File: README.md [hunk 2]
```diff
@@ -13,45 +14,6 @@ Each language, as represented by their 'xx-XX' [BCP-47 language tag](https://dev
...
-- If you're not already, switch to English
-- (NEW): For both files, copy the values for `i18nStamp` and `i18nVersion` that occur in the ones that are stored in your local storage.
-- For each file in the locales directory, copy its contents into a local storage entry of `i18next:en-US-filename`, i.e. `i18next:en-US-proton-report`
-- Reload and review
-- (NEW): When finished, remove the values you added for `i18nStamp` and `i18nVersion` before submitting your PR
-
-### Maintenance
-
-As ProtonDB continues development, new text will be added that may need translation. For a quick way to verify if a language is missing translations (or has identical ones to English), you can run: `yarn compare --locale <locale-code>`
-
```
Comment (bdefore) [inline README.md:54]: these sections should not be removed

**Comment by bdefore** [comment]
i've left three items to address above. when fixed i'm happy to merge. thanks for your help on translation @infopcgood

**Comment by bdefore** [comment]
what shall this be called in the langauge selector? is 한국어 appropriate?

**Comment by infopcgood** [comment]
> what shall this be called in the language selector? is 한국어 appropriate?

한국어 is ok.

For other changes I'll fix them ASAP.

**Comment by infopcgood** [comment]
@bdefore tried fixing it. there is a weird conflict which I couldn't fix.

**Comment by bdefore** [comment]
thanks @infopcgood for your contribution. please note it may be up to a week for this to go live since there are some unrelated changes in front of it that need testing.
